content/integrations/awsdirectconnect.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Datadog-AWS Direct Connect Integration
+integration_title: AWS Direct Connect
+kind: integration
+git_integration_title: amazon_directconnect
+doclevel: basic
+---
+
+## Overview
+
+AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
```

Comment:
Actually what I meant is we can omit this. All we need is something like "This integration collects metrics from AWS Direct Connect (e.g. <metric 1>, <metric 2>)" where metric 1 and 2 are some that users are likely to care about most.

I realize that's very austere, but it seems like it's all we need to say in this case.